### DISCUSSION TITLE
Swags Order Incomplete & Other Issues

### DISCUSSION THREAD

**Discussion by gajendrabasnet** [community/community#48129] [GitHub Education]
Earlier this month I had placed some swag orders.
It had shown me Internal Server Error at the time but later on showed 4 Duplicate orders, Two of which got shipped.
![image](https://user-images.githubusercontent.com/96136708/220315075-1a5792f2-1934-4ec5-b8d5-1fe798fbb7b4.png)
I didn't get the usual updates email I used to get with those orders.
I received them last week and every time we used to order Swags we would be given a small amount of TShirt Codes for thegithubshop.com to hand out to students.
But on both those orders invoice, the code area was empty 
![image](https://user-images.githubusercontent.com/96136708/220315334-a5f92fc4-04fe-49a4-8f8b-1458424a08da.png)
![image](https://user-images.githubusercontent.com/96136708/220315473-439269d9-e817-4023-9f86-e66e67acf0ac.png)
It was the same case in the printed invoice I got with those swags. 

It is supposed to include the t-shirt codes in there, and I used to get an email too when it was added.
But this time, neither did I get an email, nor were the codes in the invoice.
I will show you an invoice from last year when I ordered swag as an example:
![image](https://user-images.githubusercontent.com/96136708/220317444-ed46f049-22cf-4436-bd03-8429edc5ed43.png)
I have redacted the codes and my email for privacy purposes, but the point is we used to get the codes in that format.

I made a support ticket (#2013955) a week ago but like usual haven't gotten any response at all.

**Comment by gajendrabasnet** [comment]
@samus-aran Please Look into this

**Comment by mridungupta1984** [comment]
Hey! @gajendrabasnet is there any way to contact you?

**Reply by gajendrabasnet** [reply]
> Hey! @gajendrabasnet is there any way to contact you?

yeah you can email me at gajendrabasnet@gbs.edu.np

**Comment by samus-aran** [comment]
Hi @gajendrabasnet

We like to thank you for your feedback and sharing your experience of the GitHub Education Teachers swag program. Unfortunately GitHub t-shirts are currently not part of the [GitHub Teachers swag](https://education.github.com/teachers) offering and as a result, these coupons are no longer available for distribution. They had been previously but have since been discontinued. We believe the reference to t-shirt coupons in your Teacher Swag receipt/order was in error.

We understand that this news may be disappointing, and we want to express our sincere apologies for any inconvenience this may have caused. Please know that this decision was not made lightly and we’ll continue to revise our Teacher swag offerings informing our educators of any updates in the future.
